File: ConductorDemo/app/src/main/java/com/bluelinelabs/conductor/demo/util/ColorUtil.java
package com.bluelinelabs.conductor.demo.util;

import android.content.res.Resources;
import android.content.res.TypedArray;
import android.graphics.Color;

import com.bluelinelabs.conductor.demo.R;

public class ColorUtil {

    public static int getMaterialColor(Resources resources, int index) {
        TypedArray colors = resources.obtainTypedArray(R.array.mdcolor_300);

        final int returnColor = colors.getColor(index % colors.length(), Color.BLACK);

        colors.recycle();
        return returnColor;
    }

}

File: ConductorDemo/app/src/main/java/com/bluelinelabs/conductor/demo/util/ColorUtil.kt
package com.bluelinelabs.conductor.demo.util

import android.content.res.Resources
import android.graphics.Color
import com.bluelinelabs.conductor.demo.R

object ColorUtil {

    fun getMaterialColor(resources: Resources, index: Int): Int {
        val colors = resources.obtainTypedArray(R.array.mdcolor_300)

        val returnColor = colors.getColor(index % colors.length(), Color.BLACK)

        colors.recycle()
        return returnColor
    }

}
